apps/sms/views/inbox/js/inbox.js

```diff
@@ -141,6 +147,10 @@ var InboxView = {
     }
   },
 
+  /**
+   * The set of instructions to be executed before entering the inbox.
+   * @param  {Object} args Doesn't seem to be doing anything?
+   */
```

Comment:
this is default value of args = {}  , if you won't pass any argument, then args={}